### ISSUE TITLE
Test set for celeba

### ISSUE THREAD

**Issue by XuefengLi1** [primecai/Pix2NeRF#12]
Thanks for your great work!

I'm trying to test the pre-trained model on celeba dataset and cannot find how you split the dataset in the provided code or csv file. How could I get the test set for celeba as mentioned in the paper ( "We hold out 8k images as the test set. "). 

Thanks again for you work and help.

**Comment by XuefengLi1** [comment]
The authors split the dataset in dataset.py.
